### PR TITLE
chore: replace vc-quantity with vc-add-to-cart

### PR DESCRIPTION
**Pull request by ivan-kalachikov**
### Jira-link:
https://virtocommerce.atlassian.net/browse/VCST-1658
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-2.6.0-pr-1334-9b5f-9b5f974c.zip

### CODE REVIEW

File: client-app/ui-kit/components/organisms/add-to-cart/vc-add-to-cart.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <div class="vc-add-to-cart">
```
Comment (goldenmaya) [inline client-app/ui-kit/components/organisms/add-to-cart/vc-add-to-cart.vue:2]: add vc-add-to-cart--hide-button modifier

File: client-app/ui-kit/components/molecules/line-item/vc-line-item.vue [hunk 2]
```diff
@@ -358,11 +358,11 @@ watchEffect(() => {
   &__slot {
     @apply flex-none empty:hidden;
 
-    &:has(.vc-quantity, * .vc-quantity) {
+    &:has(.vc-add-to-cart, * .vc-add-to-cart) {
```
Comment (goldenmaya) [inline client-app/ui-kit/components/molecules/line-item/vc-line-item.vue:361]: vc-add-to-cart--hide-button

File: client-app/ui-kit/components/molecules/line-item/vc-line-item.vue [hunk 3]
```diff
@@ -358,11 +358,11 @@ watchEffect(() => {
...
       @apply w-[6.5rem];
     }
 
-    &:has(.vc-add-to-cart, * .vc-add-to-cart, .add-to-cart, * .add-to-cart) {
+    &:has(.add-to-cart, * .add-to-cart) {
```
Comment (goldenmaya) [inline client-app/ui-kit/components/molecules/line-item/vc-line-item.vue:365]: DO NOT remove vc-add-to-cart from here.
Add .vc-add-to-cart and NOT .vc-add-to-cart--hide-button

**Review by goldenmaya — CHANGES_REQUESTED** [review]

**Comment by goldenmaya** [comment]
Remove `VcQuantity` from `client-app/ui-kit/components/molecules/types.d.ts` as well

**Review by goldenmaya — APPROVED** [review]

**Review by NaMax66 — APPROVED** [review]

**Pull request merged**